Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
Once the Makefile issue was [reported](https://github.com/npm/npm/issues/16916) (and https://github.com/npm/npm/issues/17377 and https://github.com/npm/npm/issues/17554), upstream did in fact fix it: https://github.com/npm/npm/commit/29a5612adb622680c7edfd501c872b9f6f7bae62